metrics/prometheus/kubernetes.yaml

```diff
@@ -47,6 +47,9 @@ data:
           - source_labels: [__meta_kubernetes_pod_container_port_name]
             action: keep
             regex: kafkametrics.*
+          - source_labels: [__meta_kubernetes_pod_name]
```

Comment:
What exactly does this do? Does it just rename Prometheus _variable_  `__meta_kubernetes_pod_name` to `kubernetes_pod_name`? Or is that some other logic? If we only rename it, isn't that wasting of resources?

Reply:
From the documentation : 

> Labels starting with __ will be removed from the label set after relabeling is completed.

It means that you have a chance to get these "meta" labels in the relabeling phase because then they'll not be available (to Grafana for example). This is the reason why all the Prometheus configuration files use such way for making pod name available.

Reply:
Cool. Thanks for the explanation.